docs/remote_inference_blueprints/cohere_connector_embedding_blueprint.md

```diff
@@ -20,7 +24,7 @@ POST /_plugins/_ml/connectors/_create
       "headers": {
         "Authorization": "Bearer ${credential.cohere_key}"
       },
-      "request_body": "{ \"texts\": ${parameters.prompt}, \"truncate\": \"END\" }"
+      "request_body": "{ \"texts\": ${parameters.prompt}, \"truncate\": \"${parameters.truncate}\", \"model\": \"${parameters.model}\" }"
```

Comment:
`${parameters.prompt}` can't match the predict parameter `"texts": ["Say this is a test"]`